{"uuid": "f25bf6a0-ea9e-46d9-8db2-7ee11c2c62d2", "children": ["a1c941c9-5008-4134-9366-8ee127580664", "52f69b05-780f-4210-955f-0bea8fb6f5da"], "befores": [{"name": "record", "status": "passed", "start": 1641896242217, "stop": 1641896242222}], "afters": [{"name": "record::0", "status": "broken", "statusDetails": {"message": "TypeError: file() got an unexpected keyword argument 'file_path'\n", "trace": "  File \"C:\\Users\\navy\\AppData\\Roaming\\Python\\Python38\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"d:\\program files\\lib\\site-packages\\_pytest\\fixtures.py\", line 941, in _teardown_yield_fixture\n    next(it)\n  File \"F:\\自动化测试\\15-UI自动化测试框架 - 步骤驱动、日志、截图、录屏\\testcase\\conftest.py\", line 20, in record\n    allure.attach.file(file_path=\"tmp.mp4\",  attachment_type=allure.attachment_type.MP4)\n"}, "start": 1641896280602, "stop": 1641896280606}], "start": 1641896242217, "stop": 1641896280606}